ui/apps/platform/cypress/integration/configmanagement/ConfigurationManagement.helpers.js

```diff
@@ -255,9 +255,13 @@ export function interactAndWaitForConfigurationManagementScan(interactionCallbac
 }
 
 export function navigateToSingleEntityPage(entitiesKey) {
-    interactAndWaitForConfigurationManagementEntityPage(() => {
-        cy.get('[data-testid="side-panel"] [aria-label="External link"]').click();
-    }, entitiesKey);
+    // interactAndWaitForConfigurationManagementEntityPage(() => {
```

Comment:
upon investigation with @pedrottimark -- testing hypothesis that the gql request is caching on slow connections and thus not making the second request once the URL lands on the detail page

Reply:
Reran the tests.

This change seems worthwhile:
* with comment here
* optional comment out `function interactAndWaitForConfigurationManagementEntityPage` with reason why